Homework3

bulaevvi opened the pull request:
Все детали проекта и самооценку описал в readme.md.

**Скриншот всех DAG'ов:**
![DAG_0](https://user-images.githubusercontent.com/79763137/120930638-9e54fb00-c707-11eb-91cd-a3fabbd8b779.png)


**DAG генерации данных:**
![DAG_1](https://user-images.githubusercontent.com/79763137/120930645-a90f9000-c707-11eb-9dd8-6c04cc218a13.png)


**DAG обучения и валидации модели:**
![DAG_3](https://user-images.githubusercontent.com/79763137/120930670-be84ba00-c707-11eb-9753-8cdec529411f.png)


**DAG инференса:**
![DAG_2](https://user-images.githubusercontent.com/79763137/120930657-b167cb00-c707-11eb-81af-391d83222d74.png)

bulaevvi committed:
Add files via upload

bulaevvi committed:
Adding dags, images, test, docker-compose.yml

bulaevvi committed:
Update README.md

bulaevvi committed:
Update README.md

bulaevvi committed:
Update README.md

bulaevvi committed:
Update README.md

bulaevvi committed:
Adding fernet script

holyketzer commented on airflow_ml_dags/dags/utils.py:
```diff
@@ -0,0 +1,13 @@
+from datetime import timedelta
+
+VOLUME = 'C:/MADE/ML_in_prod/HW/airflow_ml_dags/data/:/data'
```
Хорошо придумано с этим файлом, чтобы зашарить конфиг.

имхо более корректное название константы будет `VOLUME_MOUNT`  или `VOLUME_MAPPING`

holyketzer reviewed (approved):
Самооценка соответствует проделанной работе. 

+ страдал, преодолевал и заставил всё это работать под Windows
+ бонусная активность пост в блоге https://data.mail.ru/blog/topic/view/18984/

bulaevvi committed:
Replaced e-mail in utils.py

Указал фейковый адрес электронной почты, т.к. кто-то поднял мой код и шлет алерты на мой ящик

Mikhail-M reviewed (approved):
Баллы: 42
